frontend/app/routing.js

```diff
@@ -34,11 +34,12 @@ angular.module('openproject')
   '$urlMatcherFactoryProvider',
   function($stateProvider, $urlRouterProvider, $urlMatcherFactoryProvider) {
   // TODO: Preserve #note-4 part of the URL.
-  $urlRouterProvider.when('/work_packages/{id}', function ($match) {
-    if($match.id.length === 0) {
-      return '/work_packages';
+  $urlRouterProvider.when('/work_packages/{id: [0-9]}', function ($match) {
```

Comment:
This must be changed to `.when('/work_packages/{id:[0-9]+}', ...`, or it won't match any work package.